lte/gateway/c/oai/lib/s8_proxy/s8_client_api.cpp

```diff
@@ -380,3 +430,29 @@ static void convert_proto_msg_to_itti_csr(
   }
   OAILOG_FUNC_OUT(LOG_SGW_S8);
 }
+
+void send_s8_delete_session_request(
+    imsi64_t imsi64, Imsi_t imsi, teid_t sgw_s11_teid, teid_t pgw_s5_teid,
```

Comment:
Can we just use one version of imsi in this call?

Reply:
Imsi_t is defined as arrray of digits and in s8_proxy.proto file, imsi is defined as string; So for grpc conversion, Imsi_t is required.
imsi64 is required while sending the response message i.e after receiving the grpc response from FedGW.
I can use macro, IMSI64_TO_STRING for conversion; But I thought rather conversion, we can the parameter and use it.